src/libraries/System.Net.Primitives/src/Resources/Strings.resx

```diff
@@ -114,4 +114,25 @@
   <data name="bad_endpoint_string" xml:space="preserve">
     <value>An invalid IPEndPoint was specified.</value>
   </data>
+  <data name="networkerror_unknown" xml:space="preserve">
+    <value>An unknown connection error occurred.</value>
+  </data>
+  <data name="networkerror_addressinuse" xml:space="preserve">
+    <value>The requested address is already in use.</value>
+  </data>
+  <data name="networkerror_invalidaddress" xml:space="preserve">
+    <value>The requested address is invalid.</value>
+  </data>
+  <data name="networkerror_connectionrefused" xml:space="preserve">
+    <value>No connection could be made because the target machine actively refused it.</value>
+  </data>
+  <data name="networkerror_hostnotfound" xml:space="preserve">
+    <value>No such host is known.</value>
+  </data>
+  <data name="networkerror_connectionaborted" xml:space="preserve">
+    <value>An established connection was aborted.</value>
```

Comment:
Presumably we will map `SocketError.OperationAborted` to this.

If so, this can also happen during `Accept` if you dispose the listener. So, there may not be an established connection.

If not, we need an `OperationAborted` on the enum.

Reply:
I'm always a little fuzzy on when OperationAborted happens vs ConnectionAborted. Do you understand this?

Reply:
OperationAborted is caused by CancelIoEx or closesocket.

ConnectionAborted is caused by TCP issues like retransmission failure.

Reply:
Yeah, that makes me want to add OperationAborted too.